x-pack/plugins/security_solution/public/detection_engine/rule_details_ui/pages/rule_details/execution_log_table/execution_log_columns.tsx

```diff
@@ -79,6 +83,19 @@ export const EXECUTION_LOG_COLUMNS: Array<EuiBasicTableColumn<RuleExecutionResul
     truncateText: false,
     width: '10%',
   },
+  {
+    name: i18n.COLUMN_TYPE,
+    field: 'type',
+    sortable: false,
+    width: '10%',
+    render: (value, record) => {
+      return (
+        <EuiText size="s">
+          {record.backfill ? RULE_EXECUTION_TYPE_BACKFILL : RULE_EXECUTION_TYPE_STANDARD}
+        </EuiText>
+      );
+    },
+  },
```

Comment:
Should we hide this column behind the feature flag?

Reply:
Probably it's fine, as it will be scheduled rule type, which is true